MediaBrowser.Model/Session/SessionMessageType.cs

```diff
@@ -0,0 +1,50 @@
+#pragma warning disable CS1591
```

Comment:
We should create the docs instead of disabling the warning, but this can be added in a later PR

Reply:
I just copied another enum in the Model project and this was on top of that, I'm actually not sure what it's for